options/options.js

```diff
@@ -0,0 +1,48 @@
+// Saves options to chrome.storage.sync.
+function save_options() {
+  var MB = document.getElementById('MB').value;
```

Comment:
I would like to see a better name for this. How about `"MaximumDecoderMemoryInMegabytes"` or `"DecoderMemoryLimitInMegabytes"` instead? For both the key and local variable.